src/blueapi/client/rest.py

```diff
@@ -127,10 +129,33 @@ def _request_and_deserialize(
         get_exception: Callable[[requests.Response], Exception | None] = _exception,
     ) -> T:
         url = self._url(suffix)
+        jwt_token_manager = TokenManager("blueapi-cli")
```

Comment:
Can this, or the audience, or the AuthConfig be injected into the constructor of the RestClient? Means this is configurable